addons/web/static/src/core/dropdown/dropdown.js

```diff
@@ -95,17 +94,9 @@ export class Dropdown extends Component {
 
         // Set up toggler and positioning --------------------------------------
         /** @type {string} **/
-        let position =
+        const position =
             this.props.position || (this.parentDropdown ? "right-start" : "bottom-start");
-        let [direction, variant = "middle"] = position.split("-");
-        if (localization.direction === "rtl") {
-            if (["bottom", "top"].includes(direction)) {
-                variant = variant === "start" ? "end" : "start";
-            } else {
-                direction = direction === "left" ? "right" : "left";
```

Comment:
This line is still needed for the direction caret class, am I right ?
Or maybe the directionCaretClass have to be set on the "onPositioned" callback.